draft-hardie-path-signals.md

```diff
@@ -27,15 +27,7 @@ informative:
 
 --- abstract
 
-TCP's state mechanics uses a series of well-known messages that are
-exchanged in the clear.  Because these are visible to network elements
-on the path between the two nodes setting up the transport connection,
-they are often used as signals by those network elements.  In
-transports that do not exchange these messages in the clear, on-path
-network elements lack those signals.  This document discusses the
-nature of the signals as they are seen by on-path elements and
-reflects on best practices for transports which encrypt their state
-mechanics.
+This document discusses the nature of the signals as they are seen by on-path elements, contrasting implicit and explicit signals. For example, TCP's state mechanics uses a series of well-known messages that are exchanged in the clear.  Because these are visible to network elements on the path between the two nodes setting up the transport connection, they are often used as signals by those network elements.  In transports that do not exchange these messages in the clear, on-path network elements lack those signals.  The use of explicit signals is recommended for transports which encrypt their state mechanics but a signal should only be exposed to the path only when the signal's originator intends that it be used by the network elements on the path.

```

Comment:
Your PR say "The use of explicit signals is recommended for transports which encrypt their state mechanics but a signal should only be exposed to the path only when the signal's originator intends that it be used by the network elements on the path."
What do you think of splitting this to read: This document recommends explicit signals be used by transports which encrypt their state mechanics.  It also recommends limiting such signals, so that  a signal is exposed to the path only when the signal's originator intends that it be used by the network elements on the path."

I'd replicate that change below, if you are okay with making it.

Reply:
Your revision looks fine to me.  I'm not sure if you asking me to twiddle the text.  I'm happy to do so but don't mind if you do.